apollo-backend/src/main/java/io/logz/apollo/services/DeploymentService.java

```diff
@@ -36,14 +36,22 @@ public void logDeploymentDescription(Deployment deployment) {
             DeployableVersion deployableVersion = deployableVersionDao.getDeployableVersion(deployment.getDeployableVersionId());
             Service service = serviceDao.getService(deployment.getServiceId());
             Environment env = environmentDao.getEnvironment(deployment.getEnvironmentId());
+            String committerName = deployableVersion.getCommitterName();
             MDC.put("markers", String.format("service-name:%s", service.getName()));
             MDC.put("env", env.getAvailability());
             MDC.put("region", env.getGeoRegion());
+            MDC.put("service", service.getName());
+            MDC.put("commit", deployableVersion.getGitCommitSha());
+            MDC.put("committer",  committerName != null ? committerName : "unknown");
```

Comment:
@maorshap please validate that this is what @mikeskali requested

Reply:
@maorshap please note that committerName != user that triggered deploy

Reply:
@matvey-mtn Yeah I know